bindgen/codegen/mod.rs

```diff
@@ -4787,6 +4810,7 @@ pub(crate) mod utils {
 
         let mut code = Vec::new();
 
+        writeln!(code, "#include <stdint.h>\n#include <stdbool.h>")?;
```

Comment:
I wouldn't add this here as it would break people's code that don't have these libraries or have their system path messed up (like in some windows casses I've seen). This goes in the same direction as treating the argument and return types provided by the user be just strings, if they use `bool` they have to include it in the headers themselves.

Reply:
Hm, I agree that unconditionally adding those is not a good idea, but the problem is that people can't include the headers themselves, since the wrapper C file only includes the header file where the macro is defined over which the user usually has no control and which might not have the the required headers included.

Should we add an additional option allow passing in include files for the generated C code? Or do you have a better idea?